src/app/(main)/project/[project_id]/page.tsx

```diff
@@ -108,12 +118,18 @@ export default function Chat(params: { params: { project_id: string } }) {
     newChatMutation.mutate({ formdata: fd });
   };
 
-  const samplePrompts = [
-    "How does one prepare for advanced airway management, even before the patient has arrived?",
-    "How do you assess a ICU patient's need for tracheal intubation?",
-    "What are the checks one needs to perform before intubation?",
-    "How do you do the initial assessment of the patient before oxygenation or ventilation?",
-  ];
+  const samplePrompts = useMemo(
+    () =>
+      project?.preset_questions && project?.preset_questions?.length > 0
+        ? project?.preset_questions?.sort(() => Math.random() - 0.5).slice(0, 4)
+        : [
+            "How does one prepare for advanced airway management, even before the patient has arrived?",
+            "How do you assess a ICU patient's need for tracheal intubation?",
+            "What are the checks one needs to perform before intubation?",
+            "How do you do the initial assessment of the patient before oxygenation or ventilation?",
+          ],
```

Comment:
Why should the app assume that the questions will always be about `Septic Shock`?

One of the bug that I spotted was the UI was showing this questions till the preset_questions are loaded

Reply:
@bodhish Do you want to not show any prompts if no custom prompts are defined or would you prefer to have a configurable global list of default prompts to show if a project has no custom prompts.

Reply:
Don't show anything unless the questions are set in the project

Reply:
Resolved in https://github.com/coronasafe/ayushma_fe/pull/100